File: Bioinformatics Stronghold/RNA - Transcribing DNA into RNA/RNA.R
#Importing libraries
library(readtext)
library(Biostrings)
library(clipr)

#Setting working directory
setwd("D:/_Hector/R") 

#Reading sequence txt file
file = readtext(paste0(getwd(), "/rosalind_rna.txt"))
DNAsequence = DNAString(x = file$text[[1]])

#Transcribing sequence
RNAsequence = RNAString(x = DNAsequence)

#Printing accordingly
print(RNAsequence)

#Copy to clipboard for easy submition
write_clip(as.character(RNAsequence))

File: Bioinformatics Stronghold/RNA - Transcribing DNA into RNA/RNA.rb
#Reading data from file
readData = File.open("rosalind_rna.txt")
dna_string = readData.read

for i in 0..dna_string.length()
	base = dna_string[i]
	if (base != '')
		if (base == 'T')
			print "U"
		else
			print base
		end
	end
end